src/Umbraco.Web.UI.Client/src/views/packager/views/installed.html

```diff
@@ -43,7 +55,7 @@
 
             <div class="umb-info-local-items">
 
-                <form novalidate name="uninstallForm">
+                <form novalidate name="uninstallForm" class="width-100">
```

Comment:
@Shazwazza This is not inline style a utility class we use to force elements to fill 100% width of it's parent element.